recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: c136bdc3d3a35adfbad28cb8007e94b2b9315d13cf1bcab1ae8429674d18a593
 
 build:
-  number: 2
+  number: 3
```

Comment:
I don't think this needs to be bumped. The ppc builds have been failing for a while, and the other builds haven't changed.